include/erlcloud_ecs.hrl

```diff
@@ -226,14 +226,26 @@
     protocol :: undefined | ecs_protocol()
 }).
 
+-record(ecs_network_interface, {
+    attachment_id :: undefined | binary(),
+    private_ipv4_address :: undefined | binary()
+}).
+
 -record(ecs_container, {
     container_arn :: undefined | binary(),
-    exit_code :: undefined | pos_integer(),
+    cpu :: undefined | binary(),
+    exit_code :: undefined | binary(),
```

Comment:
Is this a `binary()`? looks like an integer [here](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_Container.html#ECS-Type-Container-exitCode).